src/python/system/shell.py

```diff
@@ -350,14 +350,25 @@ def get_interpreter(file_to_execute):
   }
 
   try:
-    return interpreter_extension_map[os.path.splitext(file_to_execute)[1]]
+    extension = os.path.splitext(file_to_execute)[1]
+    interpreter = interpreter_extension_map[extension]
   except KeyError:
     return None
 
+  # TODO(mbarbella): Remove this when fuzzers have been migrated to Python 3.
+  # TODO(mbarbella): Remove the Windows platform check when bots have a
+  # python2.exe available.
+  if (invoke_with_python2 and interpreter == 'python' and
+      environment.platform() != 'WINDOWS'):
+    interpreter = 'python2'
```

Comment:
Please rebase this to after my change here: https://github.com/google/clusterfuzz/pull/1647

Reply:
+1

Reply:
Done